IronMan_MK1/tests/data/parse_question.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import json
+import re
+
+def writeJosn(filename,data):
+    file_full_name = './'+filename+'.json'
+    with open(file_full_name,'w') as file_out:
+        json.dump(data, file_out,indent=4)
+
+
+with open('input_p.txt','r') as file_in:
+    q_polar =[]
+    q_closed = []
+    q_open = []
+    question_type = 'Polar'
+    question = ['question','answer']
+    for line in file_in:
+        match_question = re.match('Question\s*(\S*).*:\s*(.*)\s',line)
+        if match_question:
+            question_type = match_question.group(1)
+            question[0] = match_question.group(2)
+            if question_type == 'Open-ended' :
+                list_element = {'Question': question[0],'Answer': None}
+                q_open.append(list_element)
+        match_answer = re.match('Answer:\s*(.*)\s', line)
+        if match_answer:
+            question[1] = match_answer.group(1)
+            if question_type == 'Polar':
+                list_element = {'Question': question[0], 'Answer': question[1]}
+                q_polar.append(list_element)
+            if question_type == 'Closed':
+                list_element = {'Question': question[0], 'Answer': question[1]}
+                q_closed.append(list_element)
+    data = {'Polar Question':q_polar,'Closed Question':q_closed,'Open_ended Question':q_open}
+
```

Comment:
- should we considered multiple possible answer in the future?

Reply:
I think for polar and closed question, there will be only one answer?